frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -61,12 +73,16 @@ export const OngoingMissionCard = ({ mission }: MissionProps): JSX.Element => {
                     missionStatus={mission.status}
                 />
             </TopContent>
-            <BottomContent>
-                <MissionStatusDisplayWithHeader status={mission.status} />
-                <MissionProgressDisplay mission={mission} />
-                <MissionRobotDisplay mission={mission} />
-                <BatteryStatusDisplay batteryLevel={mission.robot.batteryLevel} />
-            </BottomContent>
+            <StyledContent>
+                <MiddleContent>
+                    <MissionStatusDisplayWithHeader status={mission.status} />
+                    <MissionProgressDisplay mission={mission} />
+                </MiddleContent>
+                <BottomContent>
+                    <MissionRobotDisplay mission={mission} />
+                    <BatteryStatusDisplay batteryLevel={mission.robot.batteryLevel} />
+                </BottomContent>
+            </StyledContent>
```

Comment:
```suggestion
            <BottomContent>
                <MissionStatusDisplayWithHeader status={mission.status} />
                <MissionProgressDisplay mission={mission} />
                <MissionRobotDisplay mission={mission} />
                <BatteryStatusDisplay batteryLevel={mission.robot.batteryLevel} />
            </BottomContent>
```
Doing this and adding `gap: 6px;` to BottomContent will give these results for phone:
![image](https://github.com/equinor/flotilla/assets/33518988/639487a1-47d9-4adb-a78f-54fd744343dc)
![image](https://github.com/equinor/flotilla/assets/33518988/e2f685ad-a87d-48fa-8969-acfd9629d667)